container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -118,7 +118,7 @@ instance_groups:
           capabilities: []
           persistent-volumes: []
           shared-volumes: []
-          memory: 2205
+          memory: 2100
```

Comment:
Ah, we're tripling everything (677 * 3 = 2031); though in that case 2048 would be a rounder number

Reply:
Heh. Do you care enough to want me to change it ?